tests/pems/ecdsa_p256_pkcs1_cert.pem

```diff
@@ -1,13 +1,14 @@
 -----BEGIN CERTIFICATE-----
-MIIB3zCCAYWgAwIBAgIUIMTchjyl7P7qjxZ5XkZjDwvc4OcwCgYIKoZIzj0EAwIw
```

Comment:
What is the difference here?

Reply:
I regenerated it with the Subject Alternate Name set. Previously the Subject Alternate Name was blank, which made it impossible to use with s2nc + unsafe_verify_host.